openmdao/units/unit_library.ini

```diff
@@ -95,7 +95,7 @@ kat: mol/s, katal
 min: 60*s, minute
 h: 3600*s, hour
 d: 86400*s, day
-deg: (180/pi)*rad, degree
+deg: (pi/180)*rad, degree

```

Comment:
Do you multiply seconds by 3600 to get hours?

These conversion factors are read as "1 degree = FACTOR*rad".